models/enterprise_sonic/logging/sonic_logging.yaml

```diff
@@ -0,0 +1,69 @@
+---
+GENERATOR_VERSION: '1.0'
+ANSIBLE_METADATA: |
+    {
+        'metadata_version': '1.1',
+        'status': ['preview'],
+        'supported_by': 'community',
+        'license': 'Apache 2.0'
+    }
+NETWORK_OS: sonic
+RESOURCE: logging
+COPYRIGHT: Copyright 2022 Dell Inc. or its subsidiaries. All Rights Reserved
+DOCUMENTATION: |
+    module: sonic_logging
+    version_added: 2.0.0
+    short_description: Manage logging configuration on SONiC.
+    description:
+      - This module provides configuration management of logging for devices running SONiC.
+    author: "M. Zhang (@mingjunzhang2019)"
+    options:
+      config:
+        description:
+          - Specifies logging related configurations.
+        type: dict
+        suboptions:
+          remote_servers:
+            type: list
+            elements: dict
+            description:
+              - Remote logging sever configuration.
+            suboptions:
+              host:
```

Comment:
Isn't "host" a "required" value?

Reply:
Yes "host" is required value.

Reply:
Oops. I missed the fact that you had already specified the "required" attribute. 

Sorry for the noise.